lib/dom.js

```diff
@@ -96,6 +96,12 @@ declare class DataTransferItem {
 
 declare type DOMStringMap = { [key:string]: string, ... }
 
+declare interface DOMStringList {
+     length: number;
```

Comment:
This is read-only field according to docs

Reply:
I think I've corrected it by adding a + symbol.